### ISSUE TITLE
The Mandarin Chinese language cannot read the Chinese characters

### ISSUE THREAD

**Issue by Fatihyuksek** [espeak-ng/espeak-ng#499]
The Mandarin Chinese language cannot read the Chinese characters. When it comes upon a Chinese character it says the letter but instead of making the write tonal sound to indicate the meaning of that character it instead says a number in Chinese.

**Comment by jaacoppi** [comment]
```
espeak-ng -v  zh "wo3 hui4 shuo1 zho1ngwen2" -x
Full dictionary is not installed for 'zh'
 w'o21_| x'uei51_| s.w'o55_| ts.ong55w'u@35n_|
```


```
espeak-ng -v  zh "我会说中文" -x
Full dictionary is not installed for 'zh'
 w'o21_| x'uei51_| s.w'o55_| ts.'ong55_| w'u@35n_|
```

These work for me: Could you post the exact characters you used?

**Comment by Fatihyuksek** [comment]
Please use espeakng with NVDA and set the interface language to Chinese. You will see what I mean

iPhone’umdan gönderildi

jaacoppi <notifications@github.com<mailto:notifications@github.com>> şunları yazdı (2 Haz 2018 10:20):


espeak-ng -v  zh "wo3 hui4 shuo1 zho1ngwen2" -x
Full dictionary is not installed for 'zh'
 w'o21_| x'uei51_| s.w'o55_| ts.ong55w'u@35n_|


espeak-ng -v  zh "我会说中文" -x
Full dictionary is not installed for 'zh'
 w'o21_| x'uei51_| s.w'o55_| ts.'ong55_| w'u@35n_|


These work for me: Could you post the exact characters you used?

**Comment by jaacoppi** [comment]
I can confirm with NVDA 2018.1.1. 

Since the espeak command line works but NVDA doesn't this should be reported in NVDA.

**Comment by jaacoppi** [comment]
Looks like it has been fixed in current git already:

See:
https://github.com/nvaccess/nvda/issues/8033
https://github.com/nvaccess/nvda/issues/8185

**Comment by Fatihyuksek** [comment]
The problem still persists with NVDA 2018.1.1, using espeak ng Mandarin Chinese. I guess if it has been fixed in espeak ng then it is up to the developers of NVDA to update epseak to the fixed release.

Windows 10 için Posta<https://go.microsoft.com/fwlink/?LinkId=550986> ile gönderildi

________________________________
Gönderen: jaacoppi <notifications@github.com>
Gönderildi: Wednesday, June 6, 2018 3:54:59 PM
Kime: espeak-ng/espeak-ng
Bilgi: Fatihyuksek; Author
Konu: Re: [espeak-ng/espeak-ng] The Mandarin Chinese language cannot read the Chinese characters (#499)


Looks like it has been fixed in current git already:

See:
nvaccess/nvda#8033<https://github.com/nvaccess/nvda/issues/8033>
nvaccess/nvda#8185<https://github.com/nvaccess/nvda/issues/8185>

**Comment by jiangtiandao** [comment]
@Fatihyuksek 
NVDA stable version use espeak1.49.2
#338 fixed it but is not in any release version of eespeak
Since next version of NVDA use newer version of espeak-ng 
you can test with next version and determine whether this issue is related to #338

**Comment by jiangtiandao** [comment]
@Fatihyuksek 
NVDA stable version use espeak1.49.2
#338 fixed it but is not in any release version of eespeak
Since next version of NVDA use newer version of espeak-ng 
you can test with next version and determine whether this issue is related to #338

**Comment by jiangtiandao** [comment]
@Fatihyuksek 
NVDA stable version use espeak1.49.2
#338 fixed it but is not in any release version of eespeak
Since next version of NVDA use newer version of espeak-ng 
you can test with next version and determine whether this issue is related to #338

**Comment by xiongyihui** [comment]
On Ubuntu 18.04 amd64, it don't work on command line.
eSpeak NG: 1.49.2
```
$ espeak-ng --version
eSpeak NG text-to-speech: 1.49.2  Data at: /usr/lib/x86_64-linux-gnu/espeak-ng-data

$ espeak-ng -v zh "wo3 hui3 shuo1 zhong1 wen2" -x
 w'o_| s'a55n_| x'uei_| s'a55n_| s.w'o_| j'i55_| ts.'ong_| j'i55_| w'u@n_| '@r51_|

$ espeak-ng -v zh "我会说中文" -x
 w'osa55n_| x'ueisi[51_| s.w'oji55_| ts.'ongji55_| w'u@n@r51_| j'i55_|

$ espeak-ng -v zh-yue "我会说中文" -x
 N'o5_| w'ui2_| s'yu3t_| z'ung1_| m'a4n_|
```

**Comment by larry801** [comment]
@xiongyihui  try master branch 
fix to this issue isn't included in 1.49.2
